packs/ancestries/sprite.json

```
@@ -68,7 +69,7 @@
             "remaster": false,
             "title": "Pathfinder Lost Omens Ancestry Guide"
         },
-        "reach": 0,
+        "reach": 5,
```

```suggestion
        "reach": 0,
```
Don't know why that changed